packages/lesswrong/components/editor/EditTitle.tsx

```diff
@@ -70,7 +64,7 @@ const EditTitle = ({document, value, path, placeholder, updateCurrentValues, cla
   }, [document, updatePost, lastSavedTitle, flash])
 
   return <Input
-    className={classNames(classes.root, {[classes.question]: question && !isEAForum})}
+    className={classes.root}
```

Comment:
We're removing question-specific styling?

Reply:
EA Forum removed it, and I think the question-specific-styling only really made sense when there was a different Question Modal with an overall smaller UI. We changed awhile ago from nudging people towards shorter questions towards lager ones.

Reply:
(This was part of an overall sub-process of "bring our Post Page styling in line with the EA Forum tab-bar, since it made the page simpler and gave us more wiggle room for adding Dialogue specific elements)